[PowerToys]File actions menu docs (#5137)
MicrosoftDocs/windows-uwp: hub/powertoys/index.md

    :::column-end:::
:::row-end:::

### File Actions menu

:::row:::
    :::column:::

>         [![File Actions Mneu screenshot](../images/PowerToys-FileActionsMenu.png)](file-actions-menu.md)

Reviewer: ```suggestion
        [![A screenshot of the File Actions menu](../images/PowerToys-FileActionsMenu.png)](file-actions-menu.md)
```

    :::column-end:::
    :::column span="2":::

>         [File Actions menu](file-actions-menu.md) adds a seperate menu to explorer invokable via a shortcut, which adds several functionalities like generating and comparing file hashes, creating a new folder with a selection of files and much more.

Reviewer: ```suggestion
        [File Actions menu](file-actions-menu.md) adds a menu to File Explorer's context menu, invokable via a shortcut. The File Actions menu adds several options, including generating and comparing file hashes, creating a new folder with a selection of files, and much more.
```
